cmds/import_image/import_image_cmd.go

```diff
@@ -4,20 +4,46 @@ import (
 	"errors"
 	"fmt"
 
-	common "github.com/cloudfoundry-community/bosh-softlayer-tools/common"
+	"github.com/cloudfoundry-community/bosh-softlayer-tools/common"
 
+	boshlog "github.com/cloudfoundry/bosh-utils/logger"
+	boshretry "github.com/cloudfoundry/bosh-utils/retrystrategy"
 	sldatatypes "github.com/maximilien/softlayer-go/data_types"
-	softlayer "github.com/maximilien/softlayer-go/softlayer"
+	"github.com/maximilien/softlayer-go/softlayer"
+	"github.com/pivotal-golang/clock"
+)
+
+var (
```

Comment:
Let's just set lon02 as an initial DC, same as private image we created currently, we don't see any problem so far, we could enhance it later as you suggested to specify a DC list in a json file or command option. OK?